include/ruby/internal/eval.h

```diff
@@ -33,6 +33,7 @@ VALUE rb_funcallv(VALUE, ID, int, const VALUE*);
 VALUE rb_funcallv_kw(VALUE, ID, int, const VALUE*, int);
 VALUE rb_funcallv_public(VALUE, ID, int, const VALUE*);
 VALUE rb_funcallv_public_kw(VALUE, ID, int, const VALUE*, int);
+VALUE rb_funcallv_basic_kw(VALUE, ID, VALUE, int, const VALUE*, int);
```

Comment:
should not expose as C-API.

Reply:
Maybe in "internal/vm.h"?

Reply:
Functions return `Qundef` unless callable are now named as `rb_check_funcall`*.